.github/workflows/build-test-deploy.yml

```diff
@@ -755,6 +759,66 @@ jobs:
       - run: |
           cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#polkadot-node --no-update-lock-file --show-trace -L
 
+  package-statemine-node:
```

Comment:
you can use matrix for nodes

Reply:
If you do that then you can't have dependency on only one node